matrix/buffer.py

```diff
@@ -640,22 +644,19 @@ def _get_prefix_color(prefix):
 
     def _add_user_to_nicklist(self, user):
         # type: (WeechatUser) -> None
-        nick_pointer = W.nicklist_search_nick(self._ptr, "", user.nick)
-
-        if not nick_pointer:
```

Comment:
Why did you remove the conditional here?

Reply:
oops, will bring it back

Reply:
it is not required in new commit since nick_ptrs and duplicate nicks handled in separately